.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug] "
+labels: [':bug: Bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: pluginName
+    attributes:
+      label: Plugin Name and Version
+      options:
+        - "@nxrocks/spring-boot"
```

Comment:
Please add missing `"nx-"` prefix after each `@nxrocks/`  (my bad, I misspelled in in my first comment), as such:

```
- "@nxrocks/nx-spring-boot"
- "@nxrocks/nx-flutter"
- "@nxrocks/nx-quarkus"
```

Reply:
You missed this!